managed-ledger/src/test/java/org/apache/bookkeeper/test/ZooKeeperUtil.java

```diff
@@ -116,7 +115,7 @@ public void startServer(String path) throws Exception {
         // create a zookeeper client
         LOG.debug("Instantiate ZK Client");
         zkc = ZooKeeperClient.newBuilder().connectString(getZooKeeperConnectString()).build();
-        if (path != "") {
+        if (!"".equals(path)) {
```

Comment:
Why not use `StringUtils.isNotEmpty`? This utility is used very often in the project.

Reply:
I've done it

Reply:
It looks like this comment is still not addressed. Could you address this comment and rebase to master?

Reply:
This problem has been solved, see https://github.com/apache/pulsar/pull/11497#discussion_r685712017, the failure of ci build seems to have nothing to do with this pr, can you rerun the ci build?

Reply:
Sorry I didn't notice that comment. I'll rerun the CI first. BTW, you can also comment `/pulsarbot run-failure-checks` to achieve the same goal.

Reply:
Don't mind, thanks for your suggestion